RealifeTech-CoreSDK/GraphQL/GraphQLFiles/payment_createPaymentIntent.graphql

```diff
@@ -0,0 +1,30 @@
+mutation createPaymentIntent($input: PaymentIntentInput!) {
+  createPaymentIntent(input: $input) {
+    id
+    orderType
+    orderId
+    status
+    paymentSource {
+      id
+      type
+      default
+      card {
```

Comment:
I see that this `card` object is being used multiple times in your graphql statements. Is it possible to abstract this?

Reply:
Yes, I've abstracted this by using fragment. Thank you for your suggestion!